pkg/convert/go_messages.go

```diff
@@ -0,0 +1,67 @@
+package convert
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/expect-digital/translate/pkg/model"
+	"golang.org/x/text/message/pipeline"
+)
+
+func FromGoMessages(m model.Messages) ([]byte, error) {
+	pipelineMsg := messagesToPipeline(m)
+
+	msg, err := json.Marshal(pipelineMsg)
+	if err != nil {
+		return nil, fmt.Errorf("error while marshaling: %w", err)
```

Comment:
"error" is redundant. We know that this is error, better explain what you were trying to accomplish, e.g.  "marshal to go messages"